fpdf/output.py

```diff
@@ -553,7 +553,10 @@ def _add_fonts(self):
                     LOGGER.warning(
                         "Font %s is missing the following glyphs: %s",
                         fontname,
-                        ", ".join(chr(x) for x in font.missing_glyphs),
+                        ", ".join(
+                            f"{chr(x)} ({chr(x).encode('unicode-escape').decode()})"
+                            for x in font.missing_glyphs
```

Comment:
Imagine someone trying to render a lengthy chinese text with a latin font. That is going to be a multi-page log entry...

Maybe we should limit the output to the first 10 missing glyphs, or so?

Reply:
Excellent idea!
I just implemented that.